category_encoders/basen.py

```diff
@@ -178,6 +178,53 @@ def transform(self, X, override_return_df=False):
         else:
             return X.values
 
+    def inverse_transform(self, X_in):
```

Comment:
Should we use Xt as the input name to be more in line with sklearn? http://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html